docs/reference/data-streams/data-streams.asciidoc

```diff
@@ -1,67 +1,59 @@
+continuously generated time-series data.
 
-A data stream's backing indices are <<index-hidden,hidden>>.
-While all backing indices handle read requests, the most recently created
-backing index is the data stream's only write index.  A data stream only
-accepts <<docs-index_,index requests>> with `op_type` set to `create`. To update
-or delete specific documents in a data stream, submit a <<docs-delete,delete>>
-or <<docs-update,update>> API request to the backing index containing the
-document.
+Time-series data, such as logs, tends to grow over time, resulting in large
+volumes of data. While using a single {es} index for each data source is
+simpler, it is often more efficient and cost-effective to store a source's data
+across multiple, time-based indices. Multiple indices let you move indices
+containing older, less valuable data to less expensive hardware and delete
+indices when they're not longer needed, reducing overhead and storage costs.
 
-To create a data stream, set up a <<indices-templates,composable index
-template>> containing:
+A data stream is designed to give you the best of both worlds:
 
-* A name or wildcard pattern for the data stream in the `index_patterns` property.
-* A `data_stream` definition that contains the `timestamp_field` property.
-  The `timestamp_field` must be the primary timestamp field
-   for the data source. This field must be included in every
-   document indexed to the data stream.
+* The simplicity of a single, named resource you can use for requests
+  related to a data source
+* The storage, scalability, and cost-saving benefits of multiple indices 
 
-When you index one or more documents to a not-yet-existent target matching
-the template's name or pattern, {es} automatically creates the corresponding
-data stream. You can also manually create a data stream using the
-<<indices-create-data-stream,create data stream API>>. However, a composable
-template for the stream is still required.
+You can submit indexing and search requests directly to a data stream. The
+stream automatically routes the requests to a collection of hidden,
+auto-generated indices that store the source's data.
 
-You can use the <<indices-rollover-index,rollover API>> to roll a data stream
-over to a new index when the current write index meets specified criteria, such
-as a maximum age or size. A rollover creates a new backing index and updates the
-data stream's list of backing indices. This new index then becomes the stream's
-new write index. See <<rollover-data-stream-ex>>.
+You can use <<index-lifecycle-management,{ilm} ({ilm-init})>> to automate the
+management of these hidden indices. You can use {ilm-init} to spin up new
+indices, allocate indices to different hardware, delete old indices, and take
+other automatic actions based on age or size criteria you set. This lets you
+seamlessly scale your data storage based on your budget, performance,
+resiliency, and retention needs.
```

Comment:
Shall we mention the composable templates here that create/manage the data stream and backing indices settings (ie. the ILM policy) somewhere? (I think as it stands now it might imply that you need a separate policy to manage these backing indices, as opposed to the one)

Reply:
Good point. I'll add a mention here.